### DISCUSSION TITLE
Model not showing on own try

### DISCUSSION THREAD

**Discussion by Esvorst** [google/model-viewer#2006] [Q&A]
Hi all,

I can perfectly see the model that is here: https://modelviewer.dev/
But when I paste the below code snippet from the same link in a text file, save it as HTML and open it in Chrome I see the loading bar but no model is popping up.
It says 'like any other HTML', so I feel my way should work but it clearly doesn't.
Please point out my stupidity so I can have it working :)

```
<!-- Import the component -->
<script type="module" src="https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js"></script>

<!-- Use it like any other HTML element -->
<model-viewer src="shared-assets/models/Astronaut.glb" alt="A 3D model of an astronaut" auto-rotate camera-controls></model-viewer>
```

Thanks!

**Comment by felipe-chamas** [comment]
I think the `src` attribute is looking for an URL not available. If your domain is `http://localhost:8080`, `src="http://localhost:8080/shared-assets/models/Astronaut.glb"`. I believe you have some options:
- Serve the model on your domain (add the shared-assets folder to your website)
- Point to a valid model URL, such as`https://modelviewer.dev/shared-assets/models/Astronaut.glb`

**Reply by Esvorst** [reply]
You sir, are a genius. Basically all I had to do was replace src="shared-assets/models/Astronaut.glb" to src="https://modelviewer.dev/shared-assets/models/Astronaut.glb", meaning I just had to change it to the full link. Maybe they should change that in the example? Anyway, thanks alot!
